Something didn't quite work here:

```
mkdir -p -- ./libcc1
Configuring in ./libcc1
../../gcc/genhooks.c: In function 'void emit_documentation(const char*)':
../../gcc/genhooks.c:139:31: warning: ignoring return value of 'int fscanf(FILE*, const char*, ...)', declared with attribute warn_unused_result [-Wunused-result]
       fscanf (f, "%999s", buf);
                               ^
../../gcc/genhooks.c:130:3: warning: ignoring return value of 'int fscanf(FILE*, const char*, ...)', declared with attribute warn_unused_result [-Wunused-result]
   while (fscanf (f, "%*[^@]"), buf[0] = '\0',
   ^
../../gcc/genhooks.c:192:34: warning: ignoring return value of 'int fscanf(FILE*, const char*, ...)', declared with attribute warn_unused_result [-Wunused-result]
       fscanf (f, "%5[^ \n]", buf);
                                  ^
../../gcc/genhooks.c:198:31: warning: ignoring return value of 'int fscanf(FILE*, const char*, ...)', declared with attribute warn_unused_result [-Wunused-result]
       fscanf (f, "%999s", buf);
                               ^
gengtype-lex.c: In function 'int yy_get_next_buffer()':
gengtype-lex.c:2151:18: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
gengtype-lex.c:1343:20: note: in definition of macro 'YY_INPUT'
gengtype-lex.c: In function 'yy_buffer_state* yy_scan_bytes(const char*, yy_size_t)':
gengtype-lex.c:2658:19: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
GIMPLE decision tree has 1422 leafs, maximum depth 10 and a total number of 5470 nodes
removed 728 duplicate tails
GENERIC decision tree has 1422 leafs, maximum depth 10 and a total number of 5470 nodes
removed 728 duplicate tails
make[3]: *** [s-attrtab] Killed
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [all-gcc] Error 2
make[1]: *** [all] Error 2
make: *** [/build/buildroot/output/build/host-gcc-final-6.4.0/.stamp_built] Error 2
make: *** [build] Error 2
```

--- a/docs/build.md
+++ b/docs/build.md
@@ -3,6 +3,7 @@
 ## Requirements
 
 - [Docker](https://www.docker.com/)
+- 2GB Memory to build Buildroot in a container
 
 Or we provide `Vagrantfile` to create a Docker environment in your local machine.
 

